builder/hyperv/common/driver.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
```

Comment:
awesome, thanks for your quick response! I believe it should be fine, but double checking